worlds/sc2/docs/en_Starcraft 2.md

```diff
@@ -18,50 +20,74 @@ You find items by making progress in these categories:
 * Reaching milestones in the mission, such as completing part of a main objective
 * Completing challenges based on achievements in the base game, such as clearing all Zerg on Devil's Playground
 
-Except for mission completion, these categories can be disabled in the game's settings. For instance, you can disable getting items for reaching required milestones.
+Except for mission completion, these categories can be disabled in the game's settings. 
+For instance, you can disable getting items for reaching required milestones.
 
 When you receive items, they will immediately become available, even during a mission, and you will be
-notified via a text box in the top-right corner of the game screen. Item unlocks are also logged in the Archipelago client.
+notified via a text box in the top-right corner of the game screen. 
+Item unlocks are also logged in the Archipelago client.
 
-Missions are launched through the Starcraft 2 Archipelago client, through the Starcraft 2 Launcher tab. The between mission segments on the Hyperion, the Leviathan, and the Spear of Adun are not included. Additionally, metaprogression currencies such as credits and Solarite are not used.
+Missions are launched through the Starcraft 2 Archipelago client, through the Starcraft 2 Launcher tab. 
+The between mission segments on the Hyperion, the Leviathan, and the Spear of Adun are not included. 
+Additionally, metaprogression currencies such as credits and Solarite are not used.
 
 ## What is the goal of this game when randomized?
 
-The goal is to beat the final mission in the mission order. The yaml configuration file controls the mission order and how missions are shuffled.
+The goal is to beat the final mission in the mission order. 
+The yaml configuration file controls the mission order and how missions are shuffled. 
+Since the mission order determines the number of missions in a Starcraft 2 world, it can be used to customize the time 
+required to complete the world. 
+Players can choose which of the four Starcraft 2 campaigns will populate the mission order. 
```

Comment:
This phrasing makes it sound like the player only picks one campaign.  Maybe something like:

> Players can choose any combination of the four StarCraft 2 campaigns to populate the mission order.

Reply:
Not sure how you commented on an outdated version, but the spirit of the suggestion (which of the four vs combination) was applied.